source/templates/conditionals.md

```diff
@@ -10,7 +10,7 @@ We can use the `{{#if}}` helper to conditionally render a block:
 ```
 
 Handlebars will not render the block if the argument passed evaluates to
-`false`, `undefined`, `null` or `[]` (i.e., any "falsy" value).
+`false`, `undefined`, `null` or `[]` (i.e., any "falsy" value or an empty array).
```

Comment:
> any "falsy" value including an empty array

That wording, to me anyway, seems to imply that an empty array is just another falsy value in JS, which it is not.  My intention with the PR was to point out that even though an empty array in JS is truthy Handlebars will treat it as though it was falsy.

I can understand if you don't think that needs to be explained here - if that's the case I'd say its good as is.  I know its minor so I don't mean to belabor the point :smile: